rusqlite_migration/src/builder.rs

```diff
@@ -51,8 +51,7 @@ impl<'u> MigrationsBuilder<'u> {
         self
     }
 
-    /// Finalizes the builder and creates either a [`crate::Migrations`] or a
-    /// [`crate::AsyncMigrations`] instance.
+    /// Finalizes the builder and creates either a [`crate::Migrations`].
     pub fn finalize<T: FromIterator<M<'u>>>(mut self) -> T {
```

Comment:
We need to tidy this up I think, while we  are removing `AsyncMigrations`. I’ll add a commit to that MR.